### ISSUE TITLE
wrong link to uninstall-script for macos

### ISSUE THREAD

**Issue by zeitschlag** [jgm/pandoc-website#8]
[Here](http://pandoc.org/installing.html) you can find instructions on how to install/uninstall pandoc on different operating systems. For macOS, there's a link to a script, which uninstalls pandoc. [This link](https://raw.githubusercontent.com/jgm/pandoc/master/osx/uninstall-pandoc.pl) returns a "404 - Not found", the right version can be found [here](https://raw.githubusercontent.com/jgm/pandoc/master/macos/uninstall-pandoc.pl). Can you fix this, please?

**Comment by jgm** [comment]
I've fixed it in the main pandoc repo, and also manually on
the site.  The installing.txt in this repo will be updated
when we do the next release. So this can be closed.

+++ Nathan Mattes [Feb 24 17 05:29 ]:

**Comment by dkav** [comment]
The issue still hasn't been fixed on the published website.

**Comment by jgm** [comment]
Yes, it has been fixed.  I just tried it and downloaded the uninstall scritp.
You're talking about http://pandoc.org/installing.html right?
